go/oasis-test-runner/oasis/compute.go

```diff
@@ -43,6 +43,11 @@ type ComputeCfg struct {
 	Runtimes []int
 }
 
+// UpdateRuntimes updates the worker node runtimes.
+func (worker *Compute) UpdateRuntimes(runtimes []int) {
+	worker.runtimes = runtimes
```

Comment:
Do we need any locking here?

Reply:
Technically yes, but practically probably not - doesn't hurt though so can add it.